pkg/backend/aio.go

```diff
@@ -143,6 +143,10 @@ func (s *Server) ListAioControllers(_ context.Context, in *pb.ListAioControllers
 		return nil, err
 	}
 	log.Printf("Received from SPDK: %v", result)
+	if in.PageSize > 0 && int(in.PageSize) < len(result) {
```

Comment:
```
If the user specifies a negative value for page_size, the API must send an INVALID_ARGUMENT error.
```
I suppose we need to add an Issue to cover this behavior as well...

Reply:
agree. good catch. opened #298